doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -1,36 +1,51 @@
-This file contains a walkthrough of the NumPy 1.21.0 release on Linux, modified
-for building on azure and uploading to anaconda.org The commands can be copied
-into the command line, but be sure to replace 1.21.0 by the correct version.
-This should be read together with the general directions in `releasing`.
-
-
-Facility Preparation
+This is a walkthrough of the NumPy 1.21.0 release on Linux, modified for
+building with GitHub Actions and cibuildwheels and uploading to the
+`anaconda.org staging repository for NumPy <https://anaconda.org/multibuild-wheels-staging/numpy>`_.
+The commands can be copied into the command line, but be sure to replace 1.21.0
+by the correct version. This should be read together with the
+:ref:`general release guide <prepare_release>`.
+
+Facility preparation
 ====================
 
 Before beginning to make a release, use the ``*_requirements.txt`` files to
 ensure that you have the needed software. Most software can be installed with
 pip, but some will require apt-get, dnf, or whatever your system uses for
-software. Note that at this time the documentation cannot be built with Python
-3.10, for that use 3.8-3.9 instead. You will also need a GitHub personal access
-token (PAT) to push the documentation. There are a few ways to streamline things.
+software. You will also need a GitHub personal access token (PAT) to push the
+documentation. There are a few ways to streamline things:
 
 - Git can be set up to use a keyring to store your GitHub personal access token.
   Search online for the details.
 - You can use the ``keyring`` app to store the PyPI password for twine. See the
   online twine documentation for details.
 
 
-Release Preparation
+Release preparation
 ===================
 
+Add/drop Python versions
+------------------------
+
+When adding or dropping Python versions, three files need to be edited:
+
+- .github/workflows/wheels.yml  # for github cibuildwheel
+- .travis.yml  # for cibuildwheel aarch64 builds
+- setup.py  # for classifier and minimum version check.
```

Comment:
There are more CI runs in .github/workflows, as well as azure and cirrus. I am not sure that dropping/adding python versions belongs in the release walkthrough. We have some other CI documentation in the developer section.